components/CreateOrganization.js

```diff
@@ -81,31 +106,19 @@ class CreateOrganization extends React.Component {
 
   render() {
     const { LoggedInUser } = this.props;
-
-    const title = 'Create a new organization';
+    const { result, collective, status } = this.state;
+    const title = 'Create organization';
 
     return (
       <div className="CreateOrganization">
         <Header
```

Comment:
The `CreateOrganization` component is one of our oldest and part of the update means structuring it more like our newer components. We don't use `<Header>` and `<Footer>` for new components.

See at `pages/new-create-collective.js` where we wrap everything in a `<Page>` component https://github.com/opencollective/opencollective-frontend/blob/872209eec34b8567b30d4b67d7636d543e72195e/pages/new-create-collective.js#L47 which provides the top menu bar and footer so you don't have to include them.

We don't really add `classNames` anymore either - if the page needs to be broken up into significant portions, we do that by creating a new component. You've already done this correctly by creating the separate `CreateOrganizationForm` for the form 👍

And lastly, instead of using plain `<div>` we use elements like `Container`, `Flex`, or `Box`like you've done in `CreateOrganizationForm`